charts/kong/README.md

```diff
@@ -178,7 +178,7 @@ Kong can be configured via two methods:
   on Kong Ingress Controller.
   To configure and fine-tune the controller, please read the
   [Ingress Controller Parameters](#ingress-controller-parameters) section.
-- **Admin API**  
+- **Admin API**
```

Comment:
Can you revert these double white space changes? They are required to push content to the next line.

Other white space changes in the lines  below this comment are fine.

Reply:
Using whitespaces at the end of the line is a bad idea, as many people (including me) have auto-removal on save enabled in editor :)
Much better idea is just put \ at the end of the line to force line break, which I did in last comment :)

Reply:
TIL `\` is a thing in markdown. Thank you!
Can you also do this for other white spaces that are removed in the diff currently in lines above (Runtime packages section)?